### PR TITLE
[SPARK-28795][DOC][SQL] Document CREATE VIEW statement in SQL Reference

### PR DESCRIPTION
**Pull request by amanomer**
### What changes were proposed in this pull request?
Added document for CREATE VIEW command. 

### Why are the changes needed?
As a reference to syntax and examples of CREATE VIEW command.

### How was this patch tested?
Documentation update. Verified manually.

### CODE REVIEW

**Comment by dilipbiswal** [comment]
@amanomer Could you attach a screenshot of the page ?

**Review by srowen — COMMENTED** [review]
Looks pretty good. There are a load of other similar PRs to document these things right now. You might compare to the others and a) match their style or what they doc, if it seems appropriate or b) provide reviews on the others to match what you're doing, if it varies.

**Comment by amanomer** [comment]
image

**Comment by amanomer** [comment]
@gatorsmile @dilipbiswal kindly review.

**Comment by dilipbiswal** [comment]
@amanomer Can we reword the description preserving the semantics. Currently it looks similar to impala docs.

File: docs/sql-ref-syntax-ddl-create-view.md [hunk 1]
```diff
@@ -19,4 +19,53 @@ license: |
...
+    
+    create_view_clauses (order insensitive):
+        [COMMENT view_comment]
+        [TBLPROPERTIES (property_name = property_value, ...)]
+{% endhighlight %}
+
+### Examples
+{% highlight sql %}
+-- Create a global temporary view `v1` if it does not exist.
+CREATE GLOBAL TEMPORARY VIEW IF NOT EXISTS v1 AS SELECT * FROM t1;
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-ddl-create-view.md]: @amanomer Can we make this example be a little real-world like ? Like a view that contains customers for state CA or sales for month of january .. IMHO select * from t1 probably wouldn't require a view , right ?

Comment (amanomer) [reply docs/sql-ref-syntax-ddl-create-view.md]: You are definitely right. I am re-writing this. 
Thanks for your review.

**Comment by amanomer** [comment]
cc @dilipbiswal

File: docs/sql-ref-syntax-ddl-create-view.md [hunk 2]
```diff
@@ -19,4 +19,55 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Views are based on result-set of complex base SQL queries that are hard to
+maintain or understand. CREATE VIEW constructs a virtual table that have no
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-ddl-create-view.md]: back tick CREATE VIEW ?

Comment (dilipbiswal) [inline docs/sql-ref-syntax-ddl-create-view.md]: have -> has ?

File: docs/sql-ref-syntax-ddl-create-view.md [hunk 3]
```diff
@@ -19,4 +19,55 @@ license: |
...
+#### OR REPLACE
+If a view of same name already exists, it will be replaced.
+#### [GLOBAL] TEMPORARY
+TEMPORARY views are session-scoped and will be dropped when session ends 
+because it skips persisting the definition in the underlying metastore, if any.
+GLOBAL TEMPORARY views are tied to a system preserved temporary database `global_temp`.
+#### IF NOT EXISTS
+Creates a view if it does not exists.
+#### COMMENT
+Table-level and column-level comments can be specified in CREATE VIEW statement.
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-ddl-create-view.md]: is is table-level or view level ?

**Comment by dilipbiswal** [comment]
Looks good to me.
@gatorsmile @srowen for final sign off.

**Comment by amanomer** [comment]
Updated screenshot.

CreateViewSnap1
CreateViewSnap2

**Comment by amanomer** [comment]
cc @dilipbiswal @gatorsmile @srowen

File: docs/sql-ref-syntax-ddl-create-view.md [hunk 4]
```diff
@@ -19,4 +19,69 @@ license: |
...
+only change metadata. 
+
+### Syntax
+{% highlight sql %}
+CREATE [OR REPLACE] [[GLOBAL] TEMPORARY] VIEW [IF NOT EXISTS] [db_name.]view_name
+    [(column_name [COMMENT column_comment], ...) ]
+    create_view_clauses
+    AS SELECT ...;
+    
+    create_view_clauses (order insensitive):
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-ddl-create-view.md]: Can we treat `create_view_clauses` a parameter ? 
Also the query portion can be made a parameter as well ..

**Comment by dilipbiswal** [comment]
gentle ping @amanomer

**Comment by amanomer** [comment]
Working on this. Thanks for your feedback.

**Pull request closed without merging**